### PR TITLE
removes overpowered things from mining tendrils

### PR DESCRIPTION
**Pull request by ghost**
imagine killing 2 goliaths to get infinite thermal vision or a holoparasite

why did i do this if i don't even play here?

because you're all too stupid to that's why

i know the meathook got nerfed but like, still, it's pretty goddamn powerful and i'm of the opinion that mining loot should be utility-based, revolving around survivability or bolstering a miner's ability to mine, or to fight mobs, NOT to contribute to stationside combat

and the issue is a lot of lavaland loot just contributes to stationside combat and not to their jobs or helping other jobs, so this is a useful first step

and yeah i know most miners die et cetera but like, the distinction is not in the 'in practice most people are too stupid to use this' because what happens is when someone actually competent gets their mitts on this stuff they become even more unstoppable. if most miners die, yeah that means they're retarded, but just because for example most deathsquad players are idiots who'll shoot each other doesn't mean the pulse rifle isn't strong.
#### Changelog

:cl:  
rscdel: removed the meathook, clusterbuster inferno grenade (massive plasmafire), holoparasites, wisp lamp, and a wizard spell from necropolis loot. added a healing abductor gland instead so miners can get surgery to use it or contribute to science.
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Based

**Comment by Hopekz** [comment]
Someone from our codebase might need to look this over since our heathooks have been nerfed and our Holopara's are fundamentally different vs TG

**Comment by ghost** [comment]
@Hopekz i know your code honeybunch

i know meathooks got nerfed but they're still quite useful combat-wise and your holoparasites are IIRC just the stand arrow holoparasites, are they not?

**Comment by Amelia0010** [comment]
ok "adam"

**Comment by Hopekz** [comment]
> @Hopekz i know your code honeybunch
> 
> i know meathooks got nerfed but they're still quite useful combat-wise and your holoparasites are IIRC just the stand arrow holoparasites, are they not?

Well, what I mean is we would be taking balance suggestions from someone that can't even see how components interact in our game (if you don't play here with an alt). 
If you were to still play here (even on an alt) I'd take your balance suggestions more seriously.

**Comment by ghost** [comment]
@Hopekz 

I might not play here, but i'm pretty sure I know your server pretty damn well.

You have maybe five or six notorious powergamers the admin team never bans and a lot of idiots who don't know what they're doing and don't communicate. Said powergamers usually play mining, science, or botany, and sometimes genetics-if they don't roll Captain and loot those jobs anyway. 

How those components interact is going to be pretty much universal cross-servers. I'm willig to bet that miners spend most of their time mining and not doing much but on convert-antag rounds like revolution, cult, clock cult, et cetera, sometimes they show up and start pounding on groups of antags (especially cult) with shit like the Hierophant staff (or at least, before it got nerfed) or the grenade launcher from the space caravan ruin, and get away with it because, well, fair's fair they're trying to kill everyone.

And it stays in because it's only the robust people who get it and only then occasionally so it's 'situational'. 

It's not hard to guesstimate how the meathook gets used. It might not stun, but a knockdown still makes you drop your items, ti still drags someone to you, so a quick followup with a stunbaton is sufficient to basically guarantee a kill-powerful, when tazers are gone. Same with holoparasites-they might be the stand arrow ones, and thus vary highly in utility and actual powers, but regardless the ability to have someone who backs you up when you're unconscious is a powerful one and quite devastating when it comes to, say, a cultist or a ling trying to secure you for a conversion or absorption et cetera.

So yeah. I mean, I might not play here, but cross-server balancing...well, what works on one similar server works for another similar server, and you guys are basically lowpop fulp unless i'm entirely mistaken (and going by the admins i've chatted with, i'm not).

**Comment by Hopekz** [comment]
> @Hopekz
> 
> I might not play here, but i'm pretty sure I know your server pretty damn well.

Fair enough. 
I may just have an unpopular opinion for all I know just remember that we're no longer on TG's downstream and the longer you're away the less credible you seem when trying to balance our codebase.

**Comment by ghost** [comment]
You may not be TG's downstream but most of your shit is identical to TG and your playerbase hasn't changed in five years. There are some outliers-new traitor gear, clock cult and shadowlings, out of date med system and unbalanced stuns, so on and so forth, but the general gist remains the same.

**Comment by Marmio64** [comment]
good as fuck PR, meathook is indeed still OP and you can get full healing and the hand from guardians, both of which are incredibly powerful, especially healing if the holopara has good speed and potential.

**Comment by Mesalikepie** [comment]
I don't think you realize how shit most of the necropolis loot is. Today I got six necro chests... Sounds great right? 2 rods of ascepulis (useless due to pacifism), armblade (useless gone hand), 2 narsien hardsuits garbage due to slowdown, and a rune scimitar which was the ONLY good thing there which is just good for my cargo boys and the rods of ascepulis are good for team mechanics. However like you said it's all situational. Wizard spell book the only one you can get from necro chests is book of recall which is all utility so who cares. Guardians are 1/32 and random stats. Occasionally you get some good guardian other times you get explosive so eh it's random, and it's hella fun to have a guardian and due to inconsistency not op. Meat hook not gonna lie the nerf made it not the most useful. It's a mid tier necropolis chest loot due to even on station you generally wanna maintain your space from your enemies especially if they have a stun baton and you don't. Clusterbang inferno grenade is considered shit tier loot for good reason it's bad and no one uses it. Yes it can kill players en masse but due to our rules that stops it. The grenade is highly situational and only occasionally useful on station. Lastly your arguement that necro chest loot shouldn't contribute to station side combat look at toxins? Make bombs they go boom some points. Those bombs are mostly for actually doing damage to the station. Robotics what's the point of a ggyax that just helps station combat wise so shouldn't exsist by your logic. Also infinite thermals I assume you are referring to spooky lantern which isn't even pulled by you and is so fucking useful as a miner. Let's you see megafauna through walls to avoid em and are really fucking good for utility as a miner.

**Comment by Mesalikepie** [comment]
Also your disconnect from our codebase is insane. I don't think I have ever seen a captain loot science, genetics, mining or any of that. Our admin team does an excellent job clamping down on powergamers in an appropriate manner.

**Comment by ghost** [comment]
@mesalikepie

wow everything you said alone is wrong

everything you listed is untrue or outright ignorable

thanks for playing though. if you want miners to keep thermals you can let it only work on mining mobs. i let them keep eye of god because it uses up a hand slot, which is a fair trade off

the chests being useless sometimes isn't an excuse for them being roundchangers other times for minimal effort. 

'buh buh server rules make inferno grenade USELESS tho' yeah no they don't buddy, go ahead and firebomb a cult summoning with it. besides, 'the rules say you can't use this thing'-well, therefore, if that's CORRECT, it should be removed for being worthless, and if it's NOT correct, then my point is valid. which it is, because you're wrong.

'hrurr inconsistency not OP' yeah i don't care, random chance is not an excuse to be overpowered and the few times you roll something insane is game changing, which raises the question-why should something that powerful be allowed to exist? a one-in-one-millionth chance of being immortal is still a chance to be immortal for no reason.

also, what you said about the captain thing is obviously wrong per admins i've literally spoken to, and what you said about the stationside contributing to combat like toxins, robotics, wow

i don't
actually

c
a
r
e

about those, so who gives a shit?

i'm gonna nerf robotics mechs anyway but the distinction is, those are jobs meant to have an impact on the round via their work rewards. science works to produce guns. robotics works to produce mechs. that's fine. what's not fine is a miner playing a minigame off away from the station for ten minutes and coming back with a wizard spell and a holoparasite.

the tl;dr is that i don't care, even if everything you said was right, but in reality it's actually laughably wrong so i'm even justified for not caring.

**Comment by Mesalikepie** [comment]
Oh god you haven't played on our codebase in a long time have you? Eye of God just takes the eye slot not a hand slot. I don't think this PR is gonna get merged at all cause you're one ridiculous dude and have no idea what yogs is like. Up to the head coders I guess. 
tl;dr no idea who the fuck you are, never seen you play, your PR is bad, and if a head coder contributes to this discussion I will talk to them to explain my points better

**Comment by boodaliboo** [comment]
former miner main everything you removed is great being removed and then adding the gland is good compensation and also encourages miners to go to medbay

**Comment by ynot01** [comment]
> go ahead and firebomb a cult summoning with it.

this has happened to me when we were summoning

**Comment by ghost** [comment]
@Mesalikepie

Thanks for the info, removing eye of god.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -49,39 +49,31 @@
...
 				new /obj/item/disk/design_disk/modkit_disc/mob_and_turf_aoe(src)
 			else
 				new /obj/item/disk/design_disk/modkit_disc/bounty(src)
-		if(18)
+		if(17)
 			new /obj/item/warp_cube/red(src)
+		if(18)
+			new /obj/item/organ/heart/gland/heals
 		if(19)
-			new /obj/item/wisp_lantern(src)
```
Comment (fluffe9911) [inline code/modules/mining/lavaland/necropolis_chests.dm:61]: Undocumented change and mentioning the item without saying its name at the top does not count.

**Review by fluffe9911 — CHANGES_REQUESTED** [review]
Also dont remove eye of god out of spite please

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (ghost) [reply code/modules/mining/lavaland/necropolis_chests.dm:61]: ‘This change is undocumented, saying that you removed it doesn’t count’

Do you have brain damage

**Comment by ynot01** [comment]
summon item book is fun, what you got against it?

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (fluffe9911) [reply code/modules/mining/lavaland/necropolis_chests.dm:61]: Just add it to the changelog is what I meant sorry

**Comment by ghost** [comment]
@ynot01 

it is fun

it's also overpowered

wizard spells are not for normal crew. being able to summon an item to your hand any time you lose it or drop it is hugely powerful.

**Review by fluffe9911 — APPROVED** [review]

**Comment by boodaliboo** [comment]
like how your keeping the fun and unique items exclusive to that

**Comment by ghost** [comment]
@boodaliboo 

items that are unique to the necro tendril and are either thematic or don't offer a combat advantage should stay in

i'm later planning on making the hierophant staff be able to be used on the station but only to teleport, while the normal attacks remain on lavaland, rather than currently useless as it is

**Comment by Mesalikepie** [comment]
Adam... The hierophant can teleport you to a beacon on station... 
Personally I would like to say I agree necro chests need a nerf but this isn't the nerf for them.
Inferno: Needs to go that shits garbage
Meat Hook: Move to be a drop from perhaps blood drunk? Blood drunk is currently useless to kill as is
Tarot Cards: Keep in as the same for now but open up a new pr with a temporary nerf to their stat points amount till a mining stand can be made
Wisp Lantern: No change. It's downsides are massively evident with you being unable to hide or change identities easily while keeping the thermals. Thermals are also especially cheap as traitors and simple to get as genetics. Thermals are especially nice for lavaland to spot enemies lurking.
Wizard Spell Book: Change to the spell smoke. A harmless spell while still allowing fun to moments.
Eye of God: Make the wearer vulnerable rp flashes and flashbangs always when wearing (through hardsuits)

This stops the overpowered shit mostly and is a much nicer funner way to do things. 
Cheers

**Comment by ghost** [comment]
@Mesalikepie You really do seem to have brain damage.

I was talking about this for the hierophant staff. https://github.com/tgstation/tgstation/pull/52880

Current yog just has the standard hierophant staff except it can't attack on station.

Anyway, I'm not taking any of your changes, except possibly the eye of god one.

**Comment by Mesalikepie** [comment]
@Adam could you elaborate on why not?

**Comment by ghost** [comment]
@Mesalikepie I can.

**Comment by Mesalikepie** [comment]
@AdamElTablawy then I request you do so

**Comment by ghost** [comment]
No.

**Comment by wesoda25** [comment]
lmao write a book adam

**Comment by Doosteroni** [comment]
hahahhaa this PR is rich. This adam dude is the biggest prick I have ever seen on github. Listen I don't even play on  yog anymore but this suuuuuuucks!

**Review by jcatjmeow — APPROVED** [review]

**Comment by JamieD1** [comment]
Server playtime/ban status has no place on GitHub and whether or not someone plays should not be an indication of quality of a Pull request.

**Comment by githubuser4141** [comment]
What about the suits of ERT armour/power armour you can find mostly unguarded on lavaland? ERT armour competes strongly with the captain's RIG (or it's slightly better) and the t45/51/APA power armour isn't as good as the name suggests but they're still hardsuits with more armor then the HoS's suit.

**Comment by ghost** [comment]
@githubuser4141 i have no qualms with better armor for miners as a reward.

**Comment by githubuser4141** [comment]
> @githubuser4141 i have no qualms with better armor for miners as a reward.

I don't mean as a reward for killing mega/difficult fauna, the armour I'm talking about is just sitting smack bang in a empty white shuttle in the middle of nowhere. Sure you'll need luck to find it but when you do it's just a free captain's hardsuit ++ for no effort.

**Pull request merged**